pepper-apis/src/main/resources/db-changes/schema/DDP-6313-add-study-redirect-workflow.xml

```diff
@@ -11,7 +11,8 @@
                              references="workflow_state(workflow_state_id)" foreignKeyName="workflow_study_redirect_state_workflow_state_fk"/>
             </column>
             <column name="study_guid" type="varchar(10)">
-                <constraints nullable="false"/>
+            </column>
+            <column name="study_name" type="varchar(100))">
```

Comment:
There's an extra parenthesis. Also, since we have ran this liquibase file in `dev` environment already, I think liquibase migration is going to fail. I think merging this is fine since it's not in the other environments yet, but we'll need to manually fix things up in `dev`.
```suggestion
            <column name="study_name" type="varchar(100)">
```